pkgs/tools/bluetooth/bluez-tools/default.nix

```diff
@@ -0,0 +1,23 @@
+{ stdenv, autoconf, automake, glib, pkgconfig, readline, fetchgit }:
+
+stdenv.mkDerivation rec {
+  name    = "bluez-tools-${rev}";
+  rev    = "193ad6bb3db"; 
```

Comment:
You should use the timestamp of the commit instead because otherwise `nix-env -u` won't work.

Could you please point me to the documentation where you read to use the git revision hash as version number? You're not the first one. I have to fix this. :panda_face: